packages/plugin-react-native-app-state-breadcrumbs/test/app-state.test.ts

```diff
@@ -1,25 +1,29 @@
-const { describe, it, expect } = global
+import Client from '@bugsnag/core/client'
+import plugin from '../'
 
-const proxyquire = require('proxyquire').noCallThru().noPreserveCache()
+// @ts-ignore
```

Comment:
Still working this issue out - see https://github.com/DefinitelyTyped/DefinitelyTyped/issues/33311